TheBTeam.Web/Program.cs

```diff
@@ -24,8 +27,23 @@ public static void Main(string[] args)
                 .AddJsonFile("appsettings.json", false, true)
                 .Build();
 
+            var connectionString = "Server=localhost\\sqlexpress; Integrated Security=SSPI; Database=TheBTeam;";
```

Comment:
would it be possible to pull the connection string from appsettings? other than that looks good <3

Reply:
it'mean you talk about separetly database for log? Is not needed, now log is inside database thebteam as additional table with columns.

Reply:
Tzn zastanawialem sie czy nie byloby fajniej gdyby samego connection stringa bralo z settingsow, zeby w razie wu nie trzeba bylo w 2 miejscach zmieniac, bo rozumiem ze to jest tabela w bteamowej istniejacej bazie.

chyba ze bylby z tym problem to mergujemy i tyle, dziala <3

Reply:
zrobione